Documentation/bpf/progtypes.rst

```diff
@@ -323,6 +323,13 @@ talked about XDP this mode is typically implied.
   production environment either the native or offloaded mode is better
   suited and the recommended way to run XDP.
 
+..
+
+* **Best Effort**
+
+  Attempt to use Native XDP on all devices, but ignore devices without XDP
+  driver support in the kernel.
```

Comment:
small nit: Please remove this doc extension in progtypes, because this refers to kernel types, not Cilium.

Please extend https://github.com/cilium/cilium/blob/main/Documentation/network/kubernetes/kubeproxy-free.rst#loadbalancer--nodeport-xdp-acceleration instead given there the option 'best-effort' should be mentioned.